src/lib/operators/table_scan.hpp

```diff
@@ -50,13 +50,15 @@ class TableScan : public AbstractReadOnlyOperator {
 
   struct PerformanceData : public OperatorPerformanceData<AbstractOperatorPerformanceData::NoSteps> {
     std::atomic<size_t> num_chunks_with_early_out{0};
+    std::atomic<size_t> num_chunks_with_all_rows_matching{0};
     std::atomic<size_t> num_chunks_with_binary_search{0};
 
     void output_to_stream(std::ostream& stream, DescriptionMode description_mode) const override {
       OperatorPerformanceData<AbstractOperatorPerformanceData::NoSteps>::output_to_stream(stream, description_mode);
 
       const auto* const separator = description_mode == DescriptionMode::MultiLine ? "\n" : " ";
       stream << separator << "Chunks: " << num_chunks_with_early_out.load() << " skipped, ";
+      stream << separator << num_chunks_with_all_rows_matching.load() << " matched all rows,";
```

Comment:
This line might be misunderstood. It sounds like the info `input_size == output_size`, but not like there has been work skipped. I don't have an idea how to state this however. :-/

Reply:
maybe rephrase this line and the one above in `X skipped because no/all rows match?`

Reply:
"X skipped with no results, Y skipped with all matching"?

Reply:
Or maybe reuse the "matches_all/matches_none" terminology from the code? That makes it easy to find it in the code.

Reply:
Don't have a strong opinion here, I'm happy with whatever you prefer

Reply:
I think both are fine. "X skipped with no results (matches_none), Y skipped with all matching (matches_all)"?

Reply:
Reusing the terminology is a good idea, but I wouldn't directly it.

Reply:
adressed